apps/examples/testcase/le_tc/kernel/kernel_tc_main.c

```diff
@@ -100,7 +100,7 @@ int kernel_tc_main(int argc, char *argv[])
 
 #ifdef CONFIG_TC_KERNEL_LIBC_PTHREAD
 #if (!defined CONFIG_PTHREAD_MUTEX_TYPES)
-#error CONFIG_PTHREAD_MUTEX_TYPES is needed for testing LIBC_PTHREAD TC
+	/* #error CONFIG_PTHREAD_MUTEX_TYPES is needed for testing LIBC_PTHREAD TC */
```

Comment:
Regardless of `CONFIG_PTHEREAD_MUTEX_TYPES` `libc_pthread` is working right, why do we need to move this line to `libc_pthread_main`?